godtools/Views/Buttons/GTButton.swift

```diff
@@ -79,6 +79,17 @@ class GTButton: UIButton {
         self.layoutSubviews()
     }
     
+    func designAsTractModalButton() {
```

Comment:
I was thinking about that, but the Sketch only display this one (also with the background color of the modal). I will ask in the chat about this